### ISSUE TITLE
Allow users to customize priority fee

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#12116]
As as user
I want to be able to customize the priority fee
So that I can make sure my transaction is processed in a timely fashion

### Description

When sending a transaction, we want to allow the user to set the priority fee.

Because eip 1559 will change the way that fees are handled, this screen will eventually replace the current existing one.

https://www.figma.com/file/XK7vrp3pZdQ5gqQVJ7Pqgz/EIP-1559-Presentation?node-id=26%3A1793

Depending on the amount of work the slider and/or error conditions can be implemented separately.

### Technical implementation

The screen should not replace the current existing screen, but should be a toggle (the toggle mechanism can be implemented in a separate PR). 
This is because we only want to go live once the network is ready.

Currently it's not completely clear how we will be pulling in the data about base fee/maximum fee etc, so given that we have a hard deadline for this task, we can go ahead an implement the frontend separately and use mock data until things are clearer.
